bool reset
---
float32 avg_rate
